docs/Laravel.md

```diff
@@ -41,7 +41,7 @@ When generating the optimized config absolute paths will be everywhere, and they
 
 ```diff
 $app = new Bref\Bridge\Laravel\Application(
--    realpath(__DIR__ . '/../')
+-    $_ENV['APP_BASE_PATH'] ?? dirname(__DIR__)
 +    env('APP_DIR', realpath(__DIR__.'/../'))
```

Comment:
This is great thanks! However since Laravel now uses environment variables we don't have to add our own and replace this line! Instead of using `APP_DIR` we can now directly set `APP_BASE_PATH` and be done with it 🎉